{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"typealias"},{"kind":"text","text":" "},{"kind":"identifier","text":"exception_handler_array_t"},{"kind":"text","text":" = "},{"kind":"typeIdentifier","text":"UnsafeMutablePointer","preciseIdentifier":"s:Sp"},{"kind":"text","text":"<"},{"kind":"typeIdentifier","identifier":"doc:\/\/org.swift.docc\/documentation\/NIO\/exception_handler_t","preciseIdentifier":"c:@T@exception_handler_t","text":"exception_handler_t"},{"kind":"text","text":">"}],"languages":["swift"],"platforms":["macOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/nio\/exception_handler_array_t"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/org.swift.docc\/documentation\/NIO\/exception_handler_array_t","interfaceLanguage":"swift"},"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"typealias"},{"kind":"text","text":" "},{"kind":"identifier","text":"exception_handler_array_t"}],"title":"exception_handler_array_t","roleHeading":"Type Alias","role":"symbol","symbolKind":"typealias","externalID":"c:@T@exception_handler_array_t","modules":[{"name":"NIO"}],"navigatorTitle":[{"kind":"identifier","text":"exception_handler_array_t"}]},"hierarchy":{"paths":[["doc:\/\/org.swift.docc\/documentation\/NIO"]]},"references":{"doc://org.swift.docc/documentation/NIO/exception_handler_t":{"role":"symbol","title":"exception_handler_t","fragments":[{"kind":"keyword","text":"typealias"},{"kind":"text","text":" "},{"kind":"identifier","text":"exception_handler_t"}],"abstract":[],"identifier":"doc:\/\/org.swift.docc\/documentation\/NIO\/exception_handler_t","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"exception_handler_t"}],"url":"\/documentation\/nio\/exception_handler_t"},"doc://org.swift.docc/documentation/NIO/exception_handler_array_t":{"role":"symbol","title":"exception_handler_array_t","fragments":[{"kind":"keyword","text":"typealias"},{"kind":"text","text":" "},{"kind":"identifier","text":"exception_handler_array_t"}],"abstract":[],"identifier":"doc:\/\/org.swift.docc\/documentation\/NIO\/exception_handler_array_t","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"exception_handler_array_t"}],"url":"\/documentation\/nio\/exception_handler_array_t"},"doc://org.swift.docc/documentation/NIO":{"role":"collection","title":"NIO","abstract":[],"identifier":"doc:\/\/org.swift.docc\/documentation\/NIO","kind":"symbol","type":"topic","url":"\/documentation\/nio"}}}